data/json/ascii_art/gun_ascii.json

```diff
@@ -1474,53 +1474,6 @@
       "        ▀▀▀▀▀▀^~           "
     ]
   },
-  {
-    "type": "ascii_art",
```

Comment:
Are you intending to remove all of these? I didn’t see them on your list.

Reply:
`"id": "handmade_carb",`